tests/components/surepetcare/test_common.py

```diff
@@ -0,0 +1,23 @@
+"""Test helpers for the Sure Petcare integration."""
+
+from homeassistant.components.surepetcare.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def help_setup_mock_entry(hass: HomeAssistant) -> MockConfigEntry:
```

Comment:
Why not just make a fixture out of this?

Reply:
Done